openHarmony/openHarmony_file.js

```diff
@@ -208,94 +208,103 @@ Object.defineProperty($.oFolder.prototype, 'content', {
 
 
 /**
- * Lists the file names contained inside the folder.
- * @param   {string}   [filter]               Filter wildcards for the content of the folder.
- *
- * @returns {string[]}   The names of the files contained in the folder that match the filter.
+ * Enum for the type of content to retrieve from the oFolder.
+ * @enum {QFlag}
  */
-$.oFolder.prototype.listFiles = function(filter){
-    if (typeof filter === 'undefined') var filter = "*";
-
-    var _dir = new QDir;
-    _dir.setPath(this.path);
-    if (!_dir.exists) throw new Error("can't get files from folder "+this.path+" because it doesn't exist");
-    _dir.setNameFilters([filter]);
-    _dir.setFilter( QDir.Files);
-    var _files = _dir.entryList();
-
-    return _files;
+const _oFolderContentType = {
```

Comment:
in the same spirit as https://github.com/cfourney/OpenHarmony/pull/58/files?diff=split&w=0#r995167330, 

this needs to be attached to one of the classes (ex: https://github.com/cfourney/OpenHarmony/blob/master/openHarmony/openHarmony_drawing.js#L99)

It could be attached to oFolder for ex?

Reply:
Resolved with: d1dc5c70aa5ae3d7d9da9675078473c4b23ddfeb
Moved into oFolder and renamed, hopefully this means it's accessible in all scenarios.